app/assets/javascripts/discourse/app/lib/sidebar/admin-sidebar.js

```diff
@@ -216,8 +218,12 @@ export default class AdminSidebarPanel extends BaseCustomSidebarPanel {
 
   @cached
   get sections() {
-    const currentUser = getOwnerWithFallback().lookup("service:current-user");
-    const siteSettings = getOwnerWithFallback().lookup("service:site-settings");
+    const currentUser = getOwnerWithFallback(this).lookup(
+      "service:current-user"
+    );
+    const siteSettings = getOwnerWithFallback(this).lookup(
+      "service:site-settings"
+    );
```

Comment:
I believe we can remove all these changes here as well since they are mostly cosmestic now.

Reply:
I think our formatter/prettier must have changed slightly because if I change it it just changes back or complains about linting with prettier when I try to commit, so will just leave it